go/engine/login_current_device.go

```diff
@@ -67,21 +79,19 @@ func (e *LoginCurrentDevice) Run(ctx *Context) error {
 		e.G().Log.Debug("user config is nil")
 		return errNoConfig
 	}
-	if config.GetDeviceID().IsNil() {
+	did := config.GetDeviceID()
```

Comment:
can we call this variable "deviceID" or something. It took me a second to figure out what it meant :)